packages/main/src/LocalFileSystemService.ts

```diff
@@ -127,22 +127,19 @@ export const LocalFileSystemService = {
     const listener = chokidar.watch(path,{ignoreInitial:true, usePolling: os.platform()==='win32'});
     changeListeners.set(path, listener);
 
-    const updatePath = path => {
+    const updatePath = (path,type) => {
       const window = BrowserWindow.getAllWindows().find(w => !w.isDestroyed());
-      window?.webContents.send('LocalFileSystemService.updatePath', path_to_arcitect(path));
-    };
-    const updateParentPath = path => {
-      updatePath( PATH.dirname(path) );
+      window?.webContents.send('LocalFileSystemService.updatePath', [path_to_arcitect(path),type]);
     };
 
     listener
+      .on('add', path=>updatePath(path,'file_add'))
```

Comment:
Please use string unions for `file_add`, `file_rm` etc. These should be placed in a scope for general access: https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes.html#unions